aiohttp/web.py

```diff
@@ -414,12 +423,15 @@ def run_app(app, *, host=None, port=None, path=None, sock=None,
         asyncio.gather(*server_creations, loop=loop)
     )
 
+    if handle_signals:
+        signal.signal(signal.SIGTERM, sigterm_handler)
```

Comment:
https://docs.python.org/3/library/asyncio-eventloop.html#unix-signals should be used for signal registration.
Also signal subscription should be skipped on Windows